pkg/web/static/openapi.json

```diff
@@ -1275,551 +1292,824 @@
-            "format": "ULID",
-            "example": "01HV1GBK4JG9KF8RHJG2KF0B9M"
+            "format": "UUID",
+            "description": "The unique identifier of the transaction."
           },
           "source": {
             "type": "string",
-            "readOnly": true,
-            "enum": [
-              "user",
-              "gds"
-            ]
+            "description": "The source of the transaction."
           },
-          "directory_id": {
-            "type": "string",
-            "format": "UUID",
-            "example": "af0e1286-a2a1-40b1-989a-dd5b75fdfbf3",
-            "readOnly": true
-          },
-          "registered_directory": {
+          "status": {
             "type": "string",
-            "example": "trisatest.net",
-            "readOnly": true
+            "description": "The status of the transaction."
           },
-          "protocol": {
+          "counterparty": {
             "type": "string",
-            "example": "trisa",
-            "enum": [
-              "trisa",
-              "trp"
-            ]
+            "description": "The counterparty of the transaction."
           },
-          "common_name": {
+          "counterparty_id": {
             "type": "string",
-            "format": "hostname",
-            "example": "trisa.example.com"
+            "format": "ULID",
+            "description": "The unique identifier of the counterparty."
           },
-          "endpoint": {
+          "originator": {
             "type": "string",
-            "format": "uri",
-            "example": "trisa.example.com:443"
+            "description": "The originator of the transaction."
           },
-          "name": {
+          "originator_address": {
             "type": "string",
-            "example": "AliceVASP"
+            "description": "The originator's crypto address."
           },
-          "website": {
+          "beneficiary": {
             "type": "string",
-            "format": "uri"
+            "description": "The beneficiary of the transaction."
           },
-          "country": {
+          "beneficiary_address": {
             "type": "string",
-            "format": "ISO-3166-1 ALPHA-2"
+            "description": "The beneficiary's crypto address."
           },
-          "business_category": {
-            "type": "string"
-          },
-          "vasp_categories": {
-            "type": "array",
-            "items": {
-              "type": "string"
-            }
-          },
-          "verified_on": {
+          "virtual_asset": {
             "type": "string",
-            "format": "datetime"
+            "description": "The virtual asset involved in the transaction."
           },
-          "ivms101": {
-            "type": "object",
-            "description": "ivms record"
+          "amount": {
+            "type": "number",
+            "format": "double",
+            "description": "The amount of the transaction."
```

Comment:
`last_update` and `envelope_count` [appear to be missing](https://github.com/trisacrypto/envoy/blob/main/pkg/web/api/v1/transaction.go#L46-L47). Could those be added?